plugins/dai-release/src/components/TemplateHomePageComponent/TemplateHomePageComponent.tsx

```diff
@@ -0,0 +1,166 @@
+import { Content, Header, Link, LinkButton } from '@backstage/core-components';
+import { Grid, makeStyles } from '@material-ui/core';
+import {
+  ScrollableTable,
+  ScrollableTableColumn,
+} from '../DenseScrollableTable/ScrollableTable';
+import React from 'react';
+import { ReleasePopOverComponent } from '../ReleasePopOverComponent';
+import { ReleaseResponseErrorPanel } from '../ReleaseResponseErrorPanel';
+import { SearchHeaderComponent } from '../SearchHeaderComponent';
+import Typography from '@mui/material/Typography';
+import capitalize from 'lodash/capitalize';
+import { createSvgIcon } from '@mui/material/utils';
+import releaseLogoWhite from '../../assets/releaseLogoWhite.png';
+import { useTemplates } from '../../hooks';
+
+const PlusIcon = createSvgIcon(
```

Comment:
Can we move this plusIcon in any common place. which is used in densetable too.

Reply:
done